package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-payment-web",
   "description": "Digital service mock to  view scheduled payments for financial aid in the event a property subsides into mine shaft.",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Not sure how, as a team, we decide what is a patch increment and what is a minor increment. I'd have chosen minor as it's not a bug fix and is adding functionality.

Reply:
thanks, all good points. I've made the suggested changes. I've updated the code from the Bell PR to imrpove the use of string templates and remove the let.